test/mod/test_loop.py

```diff
@@ -108,6 +110,116 @@ def test_basic(tempdir):
         ctl.loop_for_fd(0)
 
 
+@pytest.mark.skipif(not TestBase.can_bind_mount(), reason="root only")
+@pytest.mark.skipif(not TestBase.ge_kernel_version(5, 8), reason="Kernel version must be >= 5.8")
+def test_configure(tempdir):
+
+    test_data = b"osbuild"
+
+    path = os.path.join(tempdir, "test.img")
+    ctl = loop.LoopControl()
+
+    assert ctl
+
+    with pytest.raises(ValueError):
+        ctl.loop_for_fd(-1)
+
+    lo, f = None, None
+    try:
+        f = open(path, "wb+")
+        f.truncate(1024)
+        f.flush()
+        #lo = ctl.loop_for_fd(f.fileno(), autoclear=True)
+        # pylint: disable=protected-access
+        ctl._check_open()
+
+        if f.fileno() < 0:
+            raise ValueError(f"Invalid file descriptor '{f.fileno()}'")
+
+        while True:
+            lo = Loop(ctl.get_unbound())
+
+            try:
+                lo.configure(f.fileno(), autoclear=True)
+            except BlockingIOError:
+                lo.clear_fd()
+                lo.close()
+                continue
+            except OSError as e:
+                lo.close()
+                # `loop_configure` returns EBUSY when the pages from the
+                # previously bound file have not been fully cleared yet.
+                if e.errno == errno.EBUSY:
+                    continue
+                raise e
```

Comment:
I wonder if it would make more sense to `assert` in this place with some message, instead of raising an exception, which will produce a traceback when running the unit test 🤔